content/influxdb/v2.0/reference/release-notes/influxdb.md

```diff
@@ -8,6 +8,115 @@ menu:
 weight: 101
 ---
 
+## v2.0.6 General Availability [2021-04-29]
+
+### Bug Fixes
+- Ensure query configuration written by `influxd upgrade `is valid.
+- Revert to nonzero defaults for `query-concurrency` and `query-queue-size`
+  configuration options to avoid validation failures when upgrading users.
```

Comment:
```suggestion
- Set `query-concurrency` and `query-queue-size` configuration option defaults
  to `0` to avoid validation failures when upgrading users.
```